### DISCUSSION TITLE
Carousel "fade" transition doesn't look good if slides have transparent background

### DISCUSSION THREAD

**Discussion by MontyHu** [twbs/bootstrap#32545] [Ideas]
I was hoping that maybe this would be improved in BS5, but, it appears to work the same way as in BS4. 

When you choose the "fade" method for slides in a carousel, if the slides have a transparent background with text, the transition looks bad because for a split second you can see the text for both slides fully on the screen overlapping each other, then the text from the previous slide abruptly disappears. Here's a screen capture showing the effect: 

![Screen Shot 2020-12-19 at 10 52 10 AM](https://user-images.githubusercontent.com/4706620/102697145-5d643b00-41e8-11eb-88f9-6f1f107ef2f1.png)

This happens because the previous slide isn't faded out while the new slide is faded in. Maybe that's not possible and is simply a limitation of browsers and CSS. When you have a solid background on the slides, the fade in looks good, but sometimes a transparent background for a slide is desired to let a textured background show through. 

Not sure if it's possible to improve this, or if there's a way to alter the carousel code to fade out one slide while the other fades in. For now I just avoid it by using the default "slide" transition, but it would be helpful to use the fade transition on slides with transparent backgrounds for people who prefer reduced motion on websites.

**Comment by XhmikosR** [comment]
@MartijnCuppens any ideas?

**Comment by itsViney** [comment]
Don't suppose anyone's found a good solution for this? I'm using a carousel inside a container that has a background image so I can't really add a background to the slide content.
